### ISSUE TITLE
feature request: dark theme

### ISSUE THREAD

**Issue by teslasmoustache** [dino/dino#507]
Please. Don't blind me.

**Comment by licaon-kter** [comment]
Use your system settings to set a dark theme.

/close

**Comment by teslasmoustache** [comment]
Yes, I already use a dark theme in Ubuntu Mate. This has no effect on Dino.

**Comment by licaon-kter** [comment]
@teslasmoustache what GTK3 theme ?

**Comment by youshitsune** [comment]
I have same problem, I use Dracula theme

**Comment by licaon-kter** [comment]
@youshitsune https://github.com/dino/dino/wiki/Frequently-asked-questions-(FAQ)#how-to-set-a-dark-theme

**Comment by poka-IT** [comment]
In Mint 21.2, I'm in dark mode and Dino stay in light mode.

This has no effect (this is not gnome):

```
gsettings set org.gnome.desktop.interface color-scheme 'prefer-dark'
```

**Comment by stereotypy** [comment]
Hi, I used this script to properly set the dark theme for Dino and other Adwaita apps: https://github.com/trinitronx/libadwaita-theme-changer/tree/add-os-theme-packages-support

However for the particular theme that I used, Dracula, there are some slight issues with formatting, the worst of which being when you mouse over messages they turn pure white and you can't read them. Not sure if this is a theme issue or because this script is a bit of a hack but thought this may help someone. This also fixed other apps that were stuck in Light mode unthemed like Pamac etc.
